Lib/test/test_buffer.py

```diff
@@ -17,7 +17,7 @@
 from itertools import permutations, product
 from random import randrange, sample, choice
 import warnings
-import sys, array, io
+import sys, array, io, os
```

Comment:
PEP 8: please write one import per line. Reorder maybe also all imports (sort them all).

Reply:
I'm maintaining this code and I'm quite happy with multiple imports per line.

Reply:
Initially I rewrote this in multiple lines. But then I seen that it required reordering virtually every line. Taking into account that changes in this PR are not strongly necessary, I prefer to do with minimal changes.